store/rootmulti/store.go

```diff
@@ -63,25 +65,27 @@
 		keysByName:   make(map[string]types.StoreKey),
 		stores:       make(map[types.StoreKey]types.CommitKVStore),
 		listeners:    make(map[types.StoreKey][]types.WriteListener),
+		mtx:          sync.RWMutex{},
```

Comment:
Seems not needed

Reply:
or do we change sync.Map to concurrent map read and map write?

Reply:
I mean don't need to init explicitly, zero value is the initial value